raiden/network/proxies/payment_channel.py

```diff
@@ -223,6 +223,7 @@ def close_light(
         """ Closes the channel using the provided balance proof. """
         self.token_network.close_light(
             channel_identifier=self.channel_identifier,
+            closing_participant=self.participant1,
```

Comment:
this isnt related with the PR issue

why was this added?

Reply:
Don't mix, be careful i put closing_participant but is not related to any other issue is just to pass the initiator address to the child function that's all.